src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,284 @@
+                    setOtherIndexDropdownValue(undefined)
+                    setOtherCallFqnDropdownValue(v.value)
+                  }
+                })
+              ])
+            ])])
+          ]),
+          otherCallFqnDropdownValue && div({ style: { marginTop: '0.25rem', marginBottom: '0.5rem', display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+            h(IdContainer, [id => h(Fragment, [
+              label({ htmlFor: id, style: { paddingRight: '0.5rem' } }, ['Shard Index:']),
+              div({ style: { flex: '1' } }, [Utils.cond(
+                [selectedCallIndex === -1, () => 'N/A (not scattered)'],
+                () => {
+                  const options = _.uniqBy('value', _.map(({ shardIndex: i }) => { return { value: i, label: i } }, otherWorkflowMetadata.calls[otherCallFqnDropdownValue]))
+                  return h(Select, { id, isSearchable: false, options, onChange: i => { setOtherIndexDropdownValue(i.value) } })
+                }
+              )])
+            ])])
+          ]),
+          otherCallFqnDropdownValue && selectedCallIndex !== undefined && !otherCallSucceeded && div({ style: { display: 'flex', alignItems: 'center', marginTop: '0.5rem' } }, [
+            icon('warning-standard', { size: 24, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+            'This call B is ineligible to call cache from because it did not succeed.'
+          ]),
+          div({ style: { display: 'flex', justifyContent: 'flex-end' } }, [
+            (otherCallFqnDropdownValue && selectedCallIndex !== undefined && otherCallSucceeded) && h(ButtonPrimary, {
+              onClick: () => { fetchDiff(otherWorkflowId, otherCallFqnDropdownValue, selectedCallIndex); setOtherCallSelected(true) }
+            }, ['Continue'])
+          ])
+        ]) :
+        'Loading workflow B\'s calls...'
+    ])
+  }
+
+  const compareDiffs = () => {
+    return h(Fragment, [
+      div({ style: { paddingBottom: '0.5rem', display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected workflow B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.codeFont } }, otherWorkflowId)
+        ]),
+        resetLink(() => resetWorkflowSelection(undefined))
+      ]),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected call B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.codeFont } }, [
+            otherCallFqnDropdownValue,
+            otherIndexDropdownValue !== undefined && breadcrumbHistoryCaret,
+            otherIndexDropdownValue !== undefined && div({ style: { marginLeft: '0.5rem', ...Style.codeFont } }, `index ${otherIndexDropdownValue}`)
```

Comment:
nit: Instead of specifying the same condition twice, this could be:
```suggestion
            otherIndexDropdownValue !== undefined && h(Fragment, [
              breadcrumbHistoryCaret,
              div({ style: { marginLeft: '0.5rem', ...Style.codeFont } }, `index ${otherIndexDropdownValue}`)
            ])
```